Task 6/routes/postgres/productsRouter.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const sequelize = require('../../sequelize/sequelize.js');
+const productsRouter =  express.Router();
+
+productsRouter.get('/', (req, res) => {
+  sequelize.Product.findAll().then(products => res.json(products));
```

Comment:
You can import { Product } from '../../sequelize/sequelize.js' to avoid repeating yourself.